test/aria/widgets/wai/input/checkbox/CheckboxJawsTestCase.js

```diff
@@ -62,7 +63,7 @@ Aria.classDefinition({
                         "Checkbox A check box not checked\nCheckbox A check box checked\nCheckbox A\nCheckbox B check box not checked\nCheckbox B\nCheckbox B check box checked\nCheckbox B\nCheckbox C check box not checked\nCheckbox C check box checked\nCheckbox C\nLast textfield\nCheckbox C check box not checked\nCheckbox B\nCheckbox B check box checked\nCheckbox B check box not checked",
                     this.end,
                     function(response) {
-                        return this.removeDuplicates(response.replace(checkedRegExp, "checked").replace(notCheckedRegExp, "not checked"));
+                        return this.removeDuplicates(response.replace(newLineBeforeCheckbox, " ").replace(checkedRegExp, "checked").replace(notCheckedRegExp, "not checked"));
```

Comment:
I usually don't like that much to use `.replace()` chain call on a single line.
I find it really more readable to have only one by line.